src/shogun/regression/LeastAngleRegressionImpl.cpp

```diff
@@ -40,23 +40,23 @@ using namespace Eigen;
 
 namespace shogun
 {
-SGMatrix<float64_t> CLeastAngleRegressionImpl::cholesky_insert_helper(SGMatrix<float64_t> X, SGMatrix<float64_t> R, VectorXd x_vec, SGVector<float64_t> active_set)
+SGMatrix<float64_t> CLeastAngleRegressionImpl::cholesky_insert_helper(Ref<MatrixXd> map_X, SGMatrix<float64_t>& R, int32_t i_max_corr, SGVector<float64_t> active_set)
```

Comment:
Develop version: [output](https://gist.github.com/Saurabh7/65c920f3813908358ded0c4ffde5b38b)
where lapack routines are used.
Other than that we need the transpose to extract columns and take ddots